ui/js/compareperf.js

```diff
@@ -201,9 +201,11 @@ perf.controller('CompareResultsCtrl', [
                     if (rsid == 'original') {
                         $scope.originalResultSetID = resultSet.id;
                         $scope.originalTimestamp = resultSet.push_timestamp;
+                        $scope.originalResultSet = resultSet;
```

Comment:
Since we're passing the resultSet object, we don't need to store originalResultSetID and originalTimestamp. Just do 

originalResultSet.push_timestamp and originalResultSet.id where you would have used originalResultSetID and originalTimestamp before.